ui/src/common/ArtistShow.js

```diff
@@ -0,0 +1,388 @@
+    },
+    mdetails: {
+      display: 'none',
+      [theme.breakpoints.down('xs')]: {
+        display: 'flex',
+        alignItems: 'center',
+        width: '7rem',
+        marginLeft: '0.5rem',
+        flex: '1',
+      },
+    },
+    mbio: {
+      display: 'none',
+      [theme.breakpoints.down('xs')]: {
+        display: 'flex',
+        marginLeft: '3%',
+        marginRight: '3%',
+        zIndex: '1',
+      },
+    },
+    content: {
+      flex: '1 0 auto',
+    },
+    cover: {
+      width: 151,
+      boxShadow: '1px 1px 20px 0px #565656',
+      borderRadius: '5px',
+    },
+    artImage: {
+      marginTop: '1rem',
+      marginLeft: '1em',
+      maxHeight: '10rem',
+      backgroundColor: 'inherit',
+      display: 'flex',
+      [theme.breakpoints.down('xs')]: {
+        marginTop: '4rem',
+        maxHeight: '7rem',
+        width: '7rem',
+      },
+    },
+    artDetail: {
+      margin: '1rem',
+      flex: '1',
+      display: 'flex',
+      minHeight: '10rem',
+      [theme.breakpoints.down('xs')]: {
+        display: 'none',
+      },
+    },
+    expand: {
+      display: 'flex',
+      padding: '0',
+      boxShadow: 'none',
+      backgroundColor: 'inherit',
+      fontSize: '0.77rem',
+      color: '#a0a0a0',
+      border: 'none',
+      '& .MuiButton-label': {
+        display: 'contents',
+      },
+      '&.MuiButton-contained:hover': {
+        boxShadow: 'none',
+        backgroundColor: 'inherit !important',
+        color: '#dbdada',
+      },
+    },
+    less: {
+      display: 'none',
+      [theme.breakpoints.down('xs')]: {
+        display: ({ link }) => (link ? 'flex' : 'none'),
+        width: '7rem',
+        flex: '1',
+        alignItems: 'flex-end',
+        padding: '0',
+        marginTop: 'auto',
+        border: 'none',
+        boxShadow: '-10px 0px 18px 5px black',
+        background: 'inherit',
+        textTransform: 'capitalize',
+        '&:hover': {
+          background: 'black',
+          boxShadow: '-10px 0px 18px 5px black',
+        },
+        '& .MuiButton-label': {
+          color: `${theme.palette.primary.main}!important`,
+        },
+      },
+    },
+    more: {
+      display: 'none',
+      [theme.breakpoints.down('xs')]: {
+        display: ({ link }) => (link ? 'flex' : 'none'),
+        width: '7rem',
+        flex: '1',
+        alignItems: 'flex-start',
+        padding: '0',
+        marginBottom: 'auto',
+        border: 'none',
+        boxShadow: '-10px 0px 18px 5px black',
+        background: 'inherit',
+        textTransform: 'capitalize',
+        '&:hover': {
+          background: 'black',
+          boxShadow: '-10px 0px 18px 5px black',
+        },
+        '& .MuiButton-label': {
+          color: `${theme.palette.primary.main}`,
+        },
+      },
+    },
+    album: {
+      marginBottom: '1em',
+    },
+  }),
+  { name: 'NDArtistPage' }
+)
+
+function ImgMediaCard({ artId, artist }) {
+  const [lastInfo, setlastInfo] = useState()
```

Comment:
Why `lastInfo` and not  `artistInfo`?

Reply:
Well, um, I was fetching the artist details(info) from LastFM, so I just named it `lastInfo`, should I change it?

Reply:
Yeah, I think it is better to give it an agnostic name. In the future we may retrieve this info from other places (ex: MusicBrainz) and may not get it using the Subsonic API. Actually it is already available in the artist resource, but it does not trigger an update as with `getArtistInfo`

Reply:
Hmm, this does have a point; well looks like it will be a breaking change for the Artist page :)
Done, I've set it to `artisteInfo`